packages/server-core/src/media/static-resource/static-resource.ts

```diff
@@ -57,4 +60,29 @@ export default (app: Application): void => {
 
   const service = app.service(staticResourcePath)
   service.hooks(hooks)
+
+  const onCRUD = (data: StaticResourceType | Paginated<StaticResourceType> | StaticResourceType[]) => {
+    const targetIds: string[] = []
```

Comment:
`userId` on static resources is not used currently. instead, we should do a query for all users who have project permissions for the project this static resource is a part of (which can be nullish, in which case we don't want to notify anyone)